### PR TITLE
Log in Log out

### PR DESCRIPTION
**Pull request by NgengChhengkim**

### CODE REVIEW

File: app/views/layouts/_header.html.erb [hunk 1]
```diff
@@ -1,12 +1,29 @@
...
+          <li class="dropdown">
+            <a href="#" class="dropdown-toggle" data-toggle="dropdown">
+              <%= current_user.username %><b class="caret"></b>
+            </a>
+            <ul class="dropdown-menu">
+              <li><%= link_to t("menus.profile"), "#" %></li>
+              <li class="divider"></li>
+              <li><%= link_to t("menus.Setting"), edit_user_registration_path %></li>
+              <li class="divider"></li>
+              <li><%= link_to t("menus.logout"), destroy_user_session_path, :method => :delete %></li>
```
Comment (HorNorin) [inline app/views/layouts/_header.html.erb]: change `:method => :delete` to `method: :delete`

File: app/views/devise/registrations/edit.html.erb [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <h2><%= t "labels.edit" %><%= resource_name.to_s.humanize %></h2>
+
+    <%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f| %>
```
Comment (RathanakSreang) [inline app/views/devise/registrations/edit.html.erb]: ` <%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: {method: :put}) do |f| %>`

**Comment by RathanakSreang** [comment]
Ok

**Comment by HorNorin** [comment]
ok

**Pull request merged**